### PR TITLE
[pbw-7473] Fixing IE11 bug that keeps replay screen up after you clic…

### PR DESCRIPTION
**Pull request by pilievOoyala**
…k the replay button. We should not be seeking to replay.

### CODE REVIEW

**Comment by pilievOoyala** [comment]
Most of the code is just wrapping the seek logic in an if statement. So it's really a two line fix and some alignment shifting.

File: src/main/js/main_html5.js [hunk 1]
```diff
@@ -455,17 +455,24 @@ require("../../../html5-common/js/utils/environment.js");
...
-      // To avoid this, save seeking information for use later.
-      // [PBW-5539] Same issue with desktop Safari when setting initialTime after video ends
-      if (OO.isAndroid || (queuedSeekRequired && !OO.isIos)) {
-        queueSeek(initialTime.value);
-      } else {
-        this.seek(initialTime.value);
+      // [PBW-7473] On replay, there is a bug on IE11 where the player won't
+      // send out some messages from the video element causing the UI to keep
+      // the replay screen up.
+      var doNotSeek = OO.isIE11Plus && hasPlayed && time === 0;
```
Comment (aeng7) [inline src/main/js/main_html5.js]: Can we do the queue Seeking that we do in the comment below (starting with "[PBW-3866]"? It sounds like these could be the same issue.

Comment (pilievOoyala) [reply src/main/js/main_html5.js]: Will do, looks like that's the better solution. yay code reviews!

**Comment by aeng7** [comment]
LGTM

**Pull request merged**